addons/website_event_track/views/event_track_views.xml

```diff
@@ -143,13 +143,12 @@
                             <group>
                                 <field name="company_id" invisible="1"/>
                                 <field name="user_id"/>
-                                <field name="partner_id" domain="['|', ('company_id', '=', company_id), ('company_id', '=', False)]"/>
                                 <field name="event_id"/>
                                 <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color'}"/>
                             </group>
                         </group>
                         <notebook>
-                            <page string="Speakers">
+                            <page string="Speaker" name="speaker">
```

Comment:
The idea behind putting "Speakers" is that you can put several names split by comma in the name field.

Reply:
OK, let me revert this part then.